pkg/fanal/analyzer/language/dotnet/nuget/proj_parser.go

```diff
@@ -0,0 +1,58 @@
+package nuget
+
+import (
+	"encoding/xml"
+
+	"golang.org/x/xerrors"
+
+	dio "github.com/aquasecurity/go-dep-parser/pkg/io"
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"github.com/aquasecurity/go-dep-parser/pkg/utils"
+)
+
+type projPackageReference struct {
+	XMLName     xml.Name `xml:"PackageReference"`
+	Version     string   `xml:"Version,attr"`
+	PackageName string   `xml:"Include,attr"`
+}
+
+type projItemGroup struct {
+	XMLName           xml.Name               `xml:"ItemGroup"`
+	PackageReferences []projPackageReference `xml:"PackageReference"`
+}
+
+type projRoot struct {
+	XMLName    xml.Name        `xml:"Project"`
+	ItemGroups []projItemGroup `xml:"ItemGroup"`
+}
+
+type Parser struct{}
```

Comment:
We usually implement a parser in go-dep-parser.

Reply:
should I create a PR  or you could transfer it there?

Reply:
Checked the go-deb-parser there is a similar pr already, it called very specific to csproj files however it could handle all dotnet sdk project files.